docs/content/nodes/archive-api-rate-limits.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Flow Archive Access API rate limits on Dapper Labs Archive nodes
+---
+
+#### Rate limits
+
+Following are the current rate limits for the [Archive Access Node gRPC API](archive-access-api) in total across all Dapper Labs Access nodes. The actual limits are enforced by each individual node. If you are using a load balanced endpoint, the limits below are the max rate, and may vary depending on how your requests are routed.
+
+Once the limit has reached, the client will receive an RPC error `ResourceExhausted` in the gRPC response.
+
+Please note, these limits only apply to the access nodes hosted by Dapper Labs. Access nodes run by other node operators will have different rate limits.
+
+##### Mainnet
+
+|                 gRPC API                             | Total request per second per client IP        |
+|:-----------------------------------------------------|:---------------------------------------------:|
+| flow.access.AccessAPI/GetAccountAtBlockHeight        |                   1                           |
+| flow.access.AccessAPI/ExecuteScriptAtBlockID         |                   1                           |
+| flow.access.AccessAPI/ExecuteScriptAtBlockHeight     |                   1                           |
+
+
+##### Testnet and Sandboxnet
+
+|                 gRPC API                             | Total request per second per IP               |
+|:-----------------------------------------------------|:---------------------------------------------:|
+| flow.access.AccessAPI/GetAccountAtLatestBlock        |                   1                           |
+| flow.access.AccessAPI/ExecuteScriptAtBlockID         |                   1                           |
+| flow.access.AccessAPI/ExecuteScriptAtBlockHeight     |                   1                           |
```

Comment:
Should these be Archive node?

Reply:
Yes! Nice catch thanks.

Reply:
Actually I synced with @koko1123 and we will need to keep the grpc api names the same here because both access and archive access api share the same protobuf definitions.

Reply:
Ok thanks - maybe then we should call out that it's intentional and why? People could get confused unless it's made clear